index.bs

```diff
@@ -1967,13 +1995,22 @@ To <dfn>obtain a randomized source response</dfn> given a [=randomized response
 
 <h3 algorithm id="computing-channel-capacity">Computing channel capacity</h3>
 
-To <dfn>compute the channel capacity of a source</dfn> given a [=randomized response output configuration=] |config| and a double |epsilon|:
-1. Let |pickRate| be the [=obtain a randomized source response pick rate|randomized response pick rate=] with |config| and |epsilon|.
-1. Let |states| be the [=obtain a set of possible trigger states|number of possible trigger states=] with |config|.
-1. If |states| is 1, return 0.
-1. If |states| is greater than the user agent's [=max trigger-state cardinality=], return an error.
-1. Let |p| be |pickRate| * (|states| - 1) / |states|.
-1. Return log2(|states|) - h(|p|) - |p| * log2(|states| - 1) where h is the binary entropy function [[BIN-ENT]].
+To <dfn>compute the channel capacity of a source</dfn> given a [=randomized response output configuration=] |config|, a double |epsilon|, a possibly null positive 32-bit integer |attributionScopeLimit|, and a possibly null positive 32-bit integer |maxNumEventStates|:
```

Comment:
I think this change would benefit from a non-normative explanation of why attribution scopes need to be accounted for in randomized-response at all, given that they do not appear in reports. In other words, if the number of destination sites doesn't affect randomized-response, why does the number of scopes?

Reply:
I see. Maybe I was confused by the renaming of algorithms here. This is an example of why updating the header validator in the PR is useful: I'd be able to see that the actual probabilities are unaffected in our tests.